frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -51,24 +54,48 @@ interface ICellProps {
   };
 }
 
-interface IHostDataColumn {
-  Header: ((props: IHeaderProps) => JSX.Element) | string;
-  Cell: (props: ICellProps) => JSX.Element;
-  id?: string;
-  title?: string;
-  accessor?: string;
-  disableHidden?: boolean;
-  disableSortBy?: boolean;
+interface IDeviceUserCellProps {
+  cell: {
+    value: IDeviceUser[];
+  };
+  row: {
+    original: IHost;
+  };
 }
 
+const condenseDeviceUsers = (users: IDeviceUser[]): string[] => {
+  if (!users?.length) {
```

Comment:
Yep, more leftovers :)

Migration to Typescript is a recent undertaking and is still a moving target. Unfortunately our TypeScript isn't a tight as it could be and there still can be a lot of unexpected `any`'s hiding out that have made me a little over cautious when it comes to the optional chaining (once burned, twice shy).

Reply:
ah, gotcha! thanks for the explanation!